pkg/tasks/c1api/full_sync.go

```diff
@@ -30,14 +30,16 @@ type fullSyncTaskHandler struct {
 	helpers fullSyncHelpers
 }
 
-func (c *fullSyncTaskHandler) sync(ctx context.Context, c1zPath string) error {
+func (c *fullSyncTaskHandler) sync(ctx context.Context, c1zPath string, debug bool) error {
 	l := ctxzap.Extract(ctx).With(zap.String("task_id", c.task.GetId()), zap.Stringer("task_type", tasks.GetType(c.task)))
 	syncer, err := sdkSync.NewSyncer(ctx, c.helpers.ConnectorClient(), sdkSync.WithC1ZPath(c1zPath), sdkSync.WithTmpDir(c.helpers.TempDir()))
 	if err != nil {
 		l.Error("failed to create syncer", zap.Error(err))
 		return err
 	}
 
+	// TODO(shackra): Change log-level to debug if `debug` is true
+
```

Comment:
Also todo: figure out how to tell the logger to save its logs to a file next to the temp c1z (initialized right above).